addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -463,12 +463,12 @@ def _invoice_constraints_peppol_en16931_ubl(self, invoice, vals):
                     # the customer’s legal entity identifier MUST be either a KVK or OIN number (schemeID 0106 or 0190)
                     'nl_r_005': _(
                         "The customer %s must have a KVK or OIN number.",
-                        vals['customer'].display_name
-                    ) if 'l10n_nl_oin' not in vals['customer']._fields or 'l10n_nl_kvk' not in vals['customer']._fields else '',
+                        vals['customer'].commercial_partner_id.display_name
+                    ) if 'l10n_nl_oin' not in vals['customer'].commercial_partner_id._fields or 'l10n_nl_kvk' not in vals['customer'].commercial_partner_id._fields else '',
                 })
 
         if vals['supplier'].country_id.code == 'NO':
-            vat = vals['supplier'].vat
+            vat = vals['supplier'].commercial_partner_id.vat
```

Comment:
is this needed ? isn't the supplier `invoice.company_id.partner_id.commercial_partner_id` ?